README.md

```diff
@@ -1 +1,48 @@
-# azure-sso-demo
\ No newline at end of file
+# azure-sso-demo
+
+This repository contains reference configurations that can create resourses on Azure using CLI and deploy cross-dc-rhsso-demo on Azure with all the required configurations.
+
+The final architecture looks like:
+
+![Architecture diagram](./scenario.jpg)
+
+
+## Prerequisites
+
+* rhel 8 with registered subscription
+* ansible >= 2.9 / python >= 3.9
+* `ansible-galaxy collection install -r requirements.yml`
+* python3-netaddr installed on the controller host `dnf install python3-netaddr`
+* Azure CLI installed on the controller host 'dnf install azure-cli'
+* All the default paramerts for Azure cloud instances are provided in 'roles/azure/defaults/main.yml'
+* We are using dynamic inventory provided  by Azure. Once the instances are created you can view it by "ansible-inventory -i inventory/myazure_rm.yml --graph"
+* We have provided two options, we can either go with managed mariadb on Azure or we can create the RHEL instances and configuire mariadb using 'mariadb' role on cross-dc-rhsso-demo.
+
+## Running on Azure cloud
+
+1. Create a var-file containing your RHN credentials:
+```
+$ cat rhn-creds.yml
+rhn_username: '<username>'
+rhn_password: '<password>'
+```
+
+2. Execute the main play:
```

Comment:
`provide_your_ssh_private_key` is not explained

Reply:
I think its self explained ? Shall I add a line as note ?

Reply:
Added:

Note: Replace `provide_your_ssh_private_key` with ssh private key file.